### PR TITLE
0006 Create contact page

### PR DESCRIPTION
**Pull request by regalcat**
__Problem / Feature__

Need a 'contact us' page that people can us to request websites or send comments to us.

__Solution__

Created a Contact Us page with a pretty web form.

__What To Test__

Make sure everything on the contact page loads and is pretty.
(The submit button doesn't do anything. Future issue.)

__Dependencies__

None

@Reviewers

@KrashLeviathan 

FYI: @KrashLeviathan @csteamengine @regalcat @MichaelRupertDev

### CODE REVIEW

**Comment by regalcat** [comment]
#16

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                <button type="submit" class="btn btn-primary">Submit</button>
+            </div>
+        </form>
+        </div>
+        
+        </div>
+        
+        </div>
+    </div>
+    </div>
```
Comment (KrashLeviathan) [inline contact.html]: These two `</div>` tags don't connect to an opening tag.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <div class="container">
+        <h4 class="page-header">Form not working?</h4>
+        <p>Email us at isuwebdevclub@gmail.com</p>
```
Comment (KrashLeviathan) [inline contact.html]: Add a hyperlink to the email address.

**Comment by KrashLeviathan** [comment]
+1 (I'll take care of the small stuff when I add the menu)

**Pull request merged**